StRoot/StMuDSTMaker/COMMON/StMuFttRawHit.h

```diff
@@ -0,0 +1,99 @@
+/***************************************************************************
+ *
+ * $Id: StMuFttRawHit.h
+ *
+ * Author: jdb, 2021
+ ***************************************************************************
+ *
+ * Description: Data class for sTGC raw hit in StMuDst
+ *
+ ***************************************************************************/
+#ifndef STMUFTTRAWHIT_H
+#define STMUFTTRAWHIT_H
+
+#include <iostream>
+#include <TObject.h>
+#include "StEnumerations.h"
+class StFttRawHit;
+
+class StMuFttRawHit : public TObject {
+public:
+    /**
+    ** @brief Default constructor.
+    **/
+    StMuFttRawHit();
+
+    StMuFttRawHit( StFttRawHit * stHit );
+
+    StMuFttRawHit(    UChar_t mSector, UChar_t mRDO, UChar_t mFEB, 
+                    UChar_t mVMM, UChar_t mChannel, UShort_t mADC, 
+                    UShort_t mBCID, Short_t mTB );
+
+    ~StMuFttRawHit() {}
+
+    void setRaw(    UChar_t mSector, UChar_t mRDO, UChar_t mFEB, 
+                    UChar_t mVMM, UChar_t mChannel, UShort_t mADC, 
+                    UShort_t mBCID, Short_t mTB );
+
+    void setMapping( UChar_t mPlane, UChar_t mQuadrant, UChar_t mRow, UChar_t mStrip, UChar_t mOrientation );
+    void set( StFttRawHit * stHit );
+
+    // consant getters
+
+    UChar_t sector() const;
+    UChar_t rdo() const;
+    UChar_t feb() const;
+    UChar_t vmm() const;
+    UChar_t channel() const;
+    UShort_t adc() const;
+    UShort_t bcid() const;
+    Short_t tb() const;
+
+    UChar_t plane() const;
+    UChar_t quadrant() const;
+    UChar_t row() const;
+    UChar_t strip() const;
+    UChar_t orientation() const;
+
+protected:
+    UChar_t mSector;
+    UChar_t mRDO;
+    UChar_t mFEB;
+    UChar_t mVMM;
+    UChar_t mChannel;
+    UShort_t mADC;
+    UShort_t mBCID;
+    Short_t mTB;  // from the trigger
+
+    // mapped information
+    UChar_t mPlane;
+    UChar_t mQuadrant;
+    UChar_t mRow;
+    UChar_t mStrip;
+    UChar_t mOrientation;
+
+    // StFttCluster *mCluster;
+    // StFttPoint   *mPoint;
+
+    ClassDef( StMuFttRawHit, 1 );
+};
+
+ostream& operator << ( ostream&, const StMuFttRawHit& hit ); // Printing operator
```

Comment:
I guess, this is how you fix the ROOT6 build:
```suggestion
std::ostream& operator << ( std::ostream&, const StMuFttRawHit& hit ); // Printing operator
```

Reply:
Daniel, would you mind taking into account Dima's suggestion and updating the request to fix the ROOT6 build issue? Let us try to get this merged to include it in fast offline?

Reply:
thanks 👍